### ISSUE TITLE
Bad Request if `fallback_locale_id` is present

### ISSUE THREAD

**Issue by mmdumi** [phrase/phrase-cli#126]
Hi,
I just saw today that my phrase cli script doesn't work any more. I tried even with the latest phrase cli executable but it's the same. I guess that the issue is server side.
Including the `fallback_locale_id` throws ERROR: 400 Bad Request
Example of pull configuration:

```
  pull:
    targets:
    - file: ./<locale_code>.lproj/Localizable.strings
      params:
        file_format: strings
        encoding: UTF-8
        include_empty_translations: true
        fallback_locale_id: 'en'
```

Removing fallback_locale_id works ok, but I really need it to be set

**Comment by jablan** [comment]
hi @mmdumi, I couldn't reproduce the problem you are seeing. I am getting error 400 for example if I use `fallback_locale_id`, but omit `include_empty_translations: true`. Unfortunately, CLI does not pass the actual 400 request error to the user, and that's something that we need to fix. I will open an internal ticket to address this issue.

In the meantime, could you please provide more info about your use case, such as ID of the project you are working with? Please send it to support@phrase.com, with a link to this issue.

Thanks

**Comment by mmdumi** [comment]
Thanks for taking a look at my problem. I just emailed with more info.

**Comment by jablan** [comment]
@mmdumi Sorry for the late feedback, we haven't received official ticket from the support team about this. I've looked deeper into the issue: turns out one gets error 400 when, for example, requests a fallback to a non-existing locale. However, this was not clear to users due to lack of feedback to the user. We deployed a small patch to our CLI client which outputs the error message received from the API. Feel free to try again, if you haven't found the problem so far.
